### PR TITLE
71787 CIE VAOS Appointment Serializer

### PR DESCRIPTION
**Pull request by kanchanasuriya**
## Summary

- *Serializer implementation for VAOS appointment data*

## Related issue(s)

- *https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/gh/department-of-veterans-affairs/va.gov-team/71787*

## Testing done

- [x] *New code is covered by unit tests*

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*
NA

## Acceptance criteria

- [x]  I added unit tests for each feature (if applicable).
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: modules/check_in/spec/serializers/vaos/vaos_appointment_data_serializer_spec.rb [hunk 1]
```diff
@@ -0,0 +1,424 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CheckIn::VAOS::VAOSAppointmentDataSerializer do
+  subject { described_class }
+
+  let(:vaos_appointment_data) do
+    {
+      data: [
```
Comment (randomsync) [inline modules/check_in/spec/serializers/vaos/vaos_appointment_data_serializer_spec.rb:10]: do we really need to put the whole payload here to test the serializer, or can we trim it?

Comment (kanchanasuriya) [reply modules/check_in/spec/serializers/vaos/vaos_appointment_data_serializer_spec.rb:10]: Thought of having the real sample data received from VAOS service

Comment (randomsync) [reply modules/check_in/spec/serializers/vaos/vaos_appointment_data_serializer_spec.rb:10]: there's almost 300 lines of the payload, that are IMHO unnecessary. I would suggest trimming it and keeping the parts that are really needed to test the serializer and trimming the rest.

Comment (kanchanasuriya) [reply modules/check_in/spec/serializers/vaos/vaos_appointment_data_serializer_spec.rb:10]: Trimmed test data further to reduce the line of code.

**Review by randomsync — APPROVED** [review]

**Pull request merged**